src/extension.ts

```diff
@@ -77,6 +78,8 @@ export function activate(context: vscode.ExtensionContext) {
     context.subscriptions.push(vscode.debug.registerDebugConfigurationProvider("edge-coreclr", {resolveDebugConfiguration}));
     context.subscriptions.push(vscode.debug.registerDebugConfigurationProvider("edge-node", {resolveDebugConfiguration}));
 
+    context.subscriptions.push(vscode.languages.registerCodeLensProvider({ pattern: `**/deployment*template.json` }, new ASAModuleUpdateCodeLensProvider()));
```

Comment:
The pattern seems not correct. Please refer "**/deployment.*.template.json,**/deployment.template.json";

Reply:
Updated `**/{deployment.*.template.json,deployment.template.json}`